about.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>Laura's Site About</title>
+  <link href="styles/normalize.css" rel="stylesheet">
+  <link href="styles/about_style.css" rel="stylesheet">
+</head>
+<body>
+  <header>
+    <nav>
+      <ul>
+        <li><a href="index.html"> Home </a></li>
+        <li><a href="portfolio.html"> Portfolio </a></li>
+        <li><a href="journal.html"> Journal </a></li>
+        <li><a href="About.html"> About </a></li>
+      </ul>
+    </nav>
+    <h1>About Laura</h1>
+
```

Comment:
One of the things I keep getting feed back on in my code at Zillow is being consistent with my use of white space.  Here removing the white space after the h1 would make for neater code